playbooks/technology_radar/artsy-tech-radar.csv

```diff
@@ -36,19 +35,25 @@ Sidekiq,adopt,tools,FALSE,
 Terraform,adopt,tools,FALSE,
 Webpack,adopt,tools,FALSE,
 Elixir,trial,languages & frameworks,TRUE,
+Next.js,trial,languages & frameworks,FALSE,"Currently trialing Next.js for our admin apps, via the pages router. See Forque, Volt 2. Some experiments have been made to migrate to Next 13's app router, which have all run into friction. See this <a href='https://artsy.slack.com/archives/C2TQ4PT8R/p1697227679293829'>Slack thread for more info</a>."
 Phoenix,trial,languages & frameworks,TRUE,
+Codepush,trial,tools,TRUE,"Microsoft codepush allows delivering updates over the air to javascript based mobile apps"
 OpsGenie,trial,tools,TRUE,
+Standard Ruby,trial,tools,TRUE,"After a few weeks living with this officially adopt."
 Unleash,trial,tools,TRUE,"Unleash offers an open-source solution for feature-flagging, gradual roll-outs, and split-testing. Official SDKs simplify consumption from multiple clients and stacks."
 VSCode pairing,trial,tools,TRUE,
-Codepush,trial,tools,TRUE,"Microsoft codepush allows delivering updates over the air to javascript based mobile apps"
 Redux,assess,languages & frameworks,FALSE,
+Vercel AI SDK,assess,languages & frameworks,TRUE,"Solid LLM abstractions in Typescript for text and data generation, with easily swappable models"
 Apollo Server,assess,platforms,TRUE,
 APIs as a product,assess,techniques,TRUE,
+Embeddings,assess,techniques,TRUE,"Dense vector mathematical representation of content, suitable for semantic search"
 GraphQL batching,assess,techniques,TRUE,
 Apollo Platform,assess,tools,TRUE,
 Datadog Synthetics,assess,tools,TRUE,
 HashiCorp Vault,assess,tools,TRUE,
 Helm,assess,tools,TRUE,
+Large Language Models,assess,tools,TRUE,Deep learning models for text generation and embeddings
```

Comment:
@joeyAghion I wasn't sure about how to use the `isNew` column. So I set all these new entries to `TRUE` — was there something else intended for this column?

Reply:
That's a Thoughtworks attribute that just renders the blip a little more visibly. I think the additions can all rightly be considered "new." We could even stop calling some old ones new, but that's for a separate update.